CONTRIBUTING.md

```diff
@@ -190,6 +224,11 @@ running:
 $ go test -v cloud.google.com/go/...
 ```
 
+You can also run a specific test suite, for instance in order to run datastore:
+``` sh
+$ go test -v cloud.google.com/go/datastore
```

Comment:
`datastore` is a different module. So, if you run this from the root of the repo, it won't use your local version of the `datastore` module. It will use the latest tagged version (you can try it by making a test fail every time locally then running this command from the repo root).

The solution is to change into a directory inside the module you want to test, then run the tests. I usually use `go test -v` or `go test -v ./...`, but you can also use the full path if you prefer.

Reply:
Just verified this. Good to know! Will update guide.